overcooked_ai_py/mdp/overcooked_env.py

```diff
@@ -134,7 +179,8 @@ def run_agents(self, agent_pair, include_final_state=False, display=False, displ
         """
         Trajectory returned will a list of state-action pairs (s_t, joint_a_t, r_t, done_t, info_t).
         """
-        assert self.cumulative_sparse_rewards == self.cumulative_shaped_rewards == 0, \
+        assert all([rew == 0 for rew in self.game_stats["cumulative_shaped_rewards_by_agent"]]) and \
+            all([rew == 0 for rew in self.game_stats["cumulative_sparse_rewards_by_agent"]]), \
             "Did not reset environment before running agents"
```

Comment:
Shouldn't it be something more like `assert not self.game_stats[key]`, that is, the game stats should be empty?

Reply:
Yeah this was a hacky way of doing it that never got changed, will update